pfio/cache/http_cache.py

```diff
@@ -0,0 +1,85 @@
+import os
+import pickle
+import time
+
+import urllib3
+
+from pfio.cache import Cache
+
+
+class HTTPCache(Cache):
+    def __init__(self, length: int, url: str, bearer_token_path=None, do_pickle=False):
```

Comment:
I think this argument spec is misleading to users; We may have to show a clear example of URL such as
```
https://cache.example.com/some/{username}/{dataset-id}/
```
to prevent applications and jobs that use very different dataset or the original data, from interfering each other by overwriting each other's data. In my example, `{username}/{dataset-id}` must be globally identical across the cache system in server side. This is because the access scope depends on the cache systems.

At least the scope or uniqueness requirement of this `url` must be clear in the docstring.

Reply:
Also, I would like this class clarified as experimental.

Reply:
Thank you for your review. I've added url's uniqueness requirement by https://github.com/pfnet/pfio/pull/282/commits/6e379fda6d2a7e7ab21206a63c36b5e8d4090fb2.